### PR TITLE
adding validation for certificate number field

### PR DESCRIPTION
**Pull request by arpakuma**
Jira ticket - https://eaflood.atlassian.net/browse/FI0-8327

Description- Validation for creating a duplicate document number

image

### CODE REVIEW

File: src/server/admin/enterCertificateNumber/index.js [hunk 1]
```diff
@@ -15,7 +19,36 @@ export const enterCertificateNumberRoutes = [
   {
     method: 'POST',
     path: '/admin/enter-certificate-number',
-    handler: (request, h) => {
+    handler: async (request, h) => {
+      const list = await getList(request)
+      if (
+        request.payload.certNumber === '' ||
+        list.findIndex(
```
Comment (Ibabalola) [inline src/server/admin/enterCertificateNumber/index.js]:
```suggestion
        list.find(
```

File: src/server/admin/enterCertificateNumber/index.js [hunk 2]
```diff
@@ -15,7 +19,36 @@ export const enterCertificateNumberRoutes = [
...
+          (listItem) => listItem.certNumber === request.payload.certNumber
+        ) !== -1
```
Comment (Ibabalola) [inline src/server/admin/enterCertificateNumber/index.js]:
```suggestion
        )
```

**Review by Ibabalola — COMMENTED** [review]
minor suggestion to use `.find()` as opposed to `.findIndex()`

File: src/server/admin/enterCertificateNumber/index.js (hunk 1 above)
Comment (arpakuma) [reply src/server/admin/enterCertificateNumber/index.js]: Thanks Isaac. updated the PR.

**Review by Ibabalola — APPROVED** [review]

**Pull request merged**